src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 40 * days;
```

Comment:
use proper names for variables - if you want to name a variable result, pick a better one :)